packages/flame/lib/src/components/component.dart

```diff
@@ -332,7 +332,6 @@ class Component with Loadable {
   ///
   /// You can return a specific sub-class of OrderedSet, like
```

Comment:
```suggestion
  /// You can return a specific sub-class of `OrderedSet`, like
```

Unrelated but bothers me. (Not sure if we can use `[]` here as that does require an import for dartdoc to resolve)